app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/model/Id.kt

```diff
@@ -1,6 +1,3 @@
 package com.github.HumanLearning2021.HumanLearningApp.model
 
-/**
- * type for arbitrary primary keys
- */
```

Comment:
please don't do that

Reply:
Sorry. Again that's cuz I made the changes from that PR manually but I should have been more careful.